File: Code_Wars/highest_and_lowest/highest_and_lowest.cs
using System;
using System.Collections.Generic;
using System.Linq;

public static class Kata
{
  public static string HighAndLow(string numbers)
  {
    
    string[] inputSplitted = numbers.Split(' ');

    List<int> listaInt = new List<int>();

    foreach (var item in inputSplitted)
    {
      
      listaInt.Add(int.Parse(item));

    }
    
    return listaInt.ToArray().Max().ToString() + " " + listaInt.ToArray().Min().ToString();
  }
}

File: Code_Wars/highest_and_lowest/highest_and_lowest.js
function highAndLow(numbers){
  
    let inputSplitted = numbers.split(" ");
  
    let arrayNumber = [];
  
    for (let index = 0; index < inputSplitted.length; index++) {
      
        arrayNumber.push(Number(inputSplitted[index]));
  
    }
    
    return Math.max.apply(0, arrayNumber) + " " + Math.min.apply(0, arrayNumber);
    
  }